primitives/state-machine/src/trie_backend.rs

```diff
@@ -724,7 +692,18 @@ pub mod tests {
 			.with_recorder(Recorder::default())
 			.build();
 		assert_eq!(trie_backend.storage(b"key").unwrap(), proving_backend.storage(b"key").unwrap());
-		assert_eq!(trie_backend.pairs(), proving_backend.pairs());
+		assert_eq!(
+			trie_backend
+				.pairs(Default::default())
```

Comment:
maybe it makes sense to create a fn `pairs_default`?

Reply:
Hmm... probably not. Maybe if it was called `pairs_all`, but then this isn't really used often enough that just doing `Default::default()` is a big deal.